stable/prometheus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: prometheus
-version: 5.6.0
-appVersion: 2.2.1
+version: 6.1.0
+appVersion: 2.2
```

Comment:
I think you want to keep this at 2.2.1 :)

Reply:
@timstoop I changed it as another commit was merged to master which added appVersion but as 2.2. You're right though this should probably be the full version so I've changed it back to 2.2.1.